modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -1,11 +1,28 @@
 // Module included in the following assemblies:
 // * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+// * post_installation_configuration/configuring-aws-outposts.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-aws-load-balancer-with-outposts_{context}"]
-= AWS Load Balancer Operator and AWS Outposts
+= Using the AWS Load Balancer Operator with AWS Outposts
 
-AWS Load Balancer Operator can work with AWS Outposts. Application Load Balancer is supported. Network Load balancer is not, so AWS Load Balancer Operator does not provision it on AWS Outposts subnets. Ingress resources must be annotated with the either outposts or regular subnets (but not both).
+You can configure the AWS Load Balancer Operator to provision an AWS Application Load Balancer in an Outpost.
+AWS Outposts does not support AWS Network Load Balancers or Classic Load Balancers.
+As a result, the AWS Load Balancer Operator cannot provision those types of load balancers in an Outpost.
+
+You can create an individual AWS Application Load Balancer either in the cloud subnet or in the Outpost subnet, but they must be separate instances.
+An Application Load Balancer in the cloud can attach to cloud-based compute nodes and an Application Load Balancer in the Outpost can attach to edge compute nodes.
+You must annotate Ingress resources with the Outpost subnet or the VPC subnet, but not both.
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have installed the AWS Load Balancer Operator and created the AWS Load Balancer Controller.
 
 .Procedure
```

Comment:
I think we should add the NodePort service example before creating the ingress resource, because this is main difference from Classic Load Balancer/Network Load Balancer. 
Application Load Balancer needs both NodePort serivice and Ingress resource.
 
```
.Example `service` resource
[source,yaml]
----
apiVersion: v1
kind: Service # <1>
metadata:
  name:  <application_name>
  namespace: <application_namespace>
spec:
  ports:
    - port: 80
      targetPort: 8080
      protocol: TCP
  type: NodePort
  selector: # <2>
    app: <application_name>
```

Reply:
Thanks @lihongan. 

The intent was to show the config difference required for the ALB in this environment. It looks like the NodePort step is part of the task "[Creating an instance of the AWS Load Balancer Controller using AWS Load Balancer Operator](https://71263--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller#nw-creating-instance-aws-load-balancer-controller_create-instance-aws-load-balancer)". Right now, I have the additional resources link pointing to general AWS LB Operator content, but I think I will update it to point to this specific task to help the user better. 

It might be a good enhancement later to simply pull that entire task into this age, with some conditions for the Outpost-specific details.

Cc: @alebedev87 and @jmanthei for awareness